src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/FrameworkRepository.cs

```diff
@@ -87,5 +88,33 @@ public Framework GetFrameworkById(string id)
 
             return document != null ? _frameworkMapping.MapToFramework(document) : null;
         }
+
+        public IEnumerable<FrameworkCodeSummary> GetAllFrameworkCodes()
+        {
+            var frameworks = GetAllFrameworks();
+
```

Comment:
Is ToList() needed here? I thought you might be able to just ToList() at the controller